docs/quickstart.rst

```diff
@@ -153,12 +137,9 @@ Now ``helloworldTimePy`` should be included in the Package list.
 You can also update or remove a SyncroSim Package from your Session using the :meth:`~pysyncrosim.session.Session.update_packages()` method or the :meth:`~pysyncrosim.session.Session.remove_packages()` method.
```

Comment:
remove text for `update_packages()`, change text for `remove_packages()` to `uninstall_packages()`